configs/14.0/deb/monolithic/bionic_dh_change_dest.patch

```diff
@@ -1 +0,0 @@
-../../../13.0/deb/monolithic/bionic_dh_change_dest.patch
```

Comment:
Can you put the link-breaking in a separate commit, so the actual changes are easier to review?
(Exactly what I _didn't_ do in a recent commit of my own. ;-)

Reply:
@ThinkOpenly that is exactly what 9dff474a91 does. The script for Ubuntu20 (focal) was already a hard file, so I only had to break the links for the other two. This commit breaks the link and just copies the contents without any changes.

Reply:
Sorry, you are correct, of course. I was looking at the net changes, not the individual commits. Thanks.